packages/x-tailwindcss/demo/tailwind.config.ts

```diff
@@ -2,26 +2,5 @@ import xTailwindCss from '../src/x-tailwind-plugin/plugin';
 
 export default {
   content: ['./index.html', './**/*.vue'],
-  prefix: 'x-',
-  important: true,
-  theme: {
-    extend: {}
-  },
-  plugins: [
-    xTailwindCss({
-      components() {
-        return {};
-      },
-      utilities() {
-        return {};
-      },
-      dynamicUtilities() {
-        return {};
-      },
-      dynamicComponents() {
-        return {};
-      },
-      theme: {}
-    })
-  ]
```

Comment:
Hi, if I am not wrong, maybe we should keep this part to be able to access to components() within the setups 👀

Reply:
I think I'm out of context and I didn't probably understand that, 
You mean you need this block to do setups? for what purpose ? Could you give me an example ? 🙏🏽